core/src/main/java/org/apache/iceberg/DataFilesTable.java

```diff
@@ -102,8 +102,8 @@ protected TableScan newRefinedScan(TableOperations ops, Table table, Schema sche
     }
 
     @Override
-    protected long targetSplitSize(TableOperations ops) {
-      return ops.current().propertyAsLong(
+    public long targetSplitSize() {
+      return tableOps().current().propertyAsLong(
```

Comment:
Why not just access `ops` directly like before?

Reply:
Before `ops` is passed in as parameter, but it seems we always pass `BaseTableScan.ops` as the argument for it. So instead I removed the parameter and just directly call `tableOps()` which does the same thing. 

Do you think we'll ever need to pass different `TableOperations` other than ``BaseTableScan.ops` for this method?

Reply:
No, my concern is that if `ops` is available as an instance field it is probably better to access it that way rather than through a method call.